### ISSUE TITLE
Button not accessible - login button?

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#7780]
**What**

[Pa11y dashboard](https://dash.wellcomecollection.org/pa11y/index.html) reports:
"error: This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined."

**Where**

See https://dash.wellcomecollection.org/pa11y/index.html for where this issue appears, but e.g. 
- https://wellcomecollection.org/
- https://wellcomecollection.org/visit-us
-  https://wellcomecollection.org/whats-on 
etc

### TODO
- [x] Make button accessible

**Comment by melanierogan** [comment]
<img width="1424" alt="Screenshot 2022-03-09 at 16 08 08" src="https://user-images.githubusercontent.com/16557524/157481332-e7f7b896-2ca3-4d54-8dfd-b0b04d87a282.png">

**Comment by davidpmccormick** [comment]
Moving this back to In progress while I figure out what's what with pa11y
